types/proxy-agent.d.ts

```diff
@@ -24,5 +24,6 @@ declare namespace ProxyAgent {
     requestTls?: buildConnector.BuildOptions;
     proxyTls?: buildConnector.BuildOptions;
     clientFactory?(origin: URL, opts: object): Dispatcher;
+    tunnelProxy?: boolean;
```

Comment:
Ditto

Reply:
this is technically still used

Reply:
Oh sorry, I misread it, disregard my comment